openprescribing/frontend/tests/commands/test_import_measures.py

```diff
@@ -109,10 +110,120 @@ def test_reconstructor_called_when_no_measures_specified(self, conn):
 class BigqueryFunctionalTests(TestCase):
     @classmethod
     def setUpTestData(cls):
-        cls.env = patch.dict(
```

Comment:
ISTR this was necessary because we were calling a management command which didn't know it should be working on the test database; perhaps via a shell which we're no longer doing?  Anyway, if it all seems to work it's no longer necessary - just wanted to mention it.

Reply:
Pretty sure that by the time we try patching the environment, the settings have already been loaded and `DATABASES` will already have been set.

Reply:
Not if subsequent steps call via a subshell? Anyway, no matter. Either way I clearly remember this being necessary at some point, and taking be ages to get right, so just wanted to wave a flag to make it ring a bell if necessary in the future.

Reply:
> Not if subsequent steps call via a subshell?

The patching here doesn't change the environment -- it just changes what `os.environ` returns.